libs/hdf-converters/src/msft-secure-score-mapper.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  SecureScore,
+  ControlScore,
+  SecureScoreControlProfile
+} from '@microsoft/microsoft-graph-types';
+import {ExecJSON} from 'inspecjs';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+
+export class MsftSecureScoreMapper extends BaseConverter {
+  withRaw: boolean;
+  profiles: SecureScoreControlProfile[];
+
+  private getProfiles(controlName: string): SecureScoreControlProfile[] {
+    return this.profiles.filter((profile) => profile.id === controlName);
+  }
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion
+    },
+    version: HeimdallToolsVersion,
+    statistics: {},
+    profiles: [
+      {
+        name: 'Microsoft Secure Score Scan',
+        title: {
+          transformer: (data: SecureScore) =>
+            `Azure Secure Score report - TenantID: ${data.azureTenantId}`
```

Comment:
what's the difference between the azuretenantid and the id?

Reply:
azureTenantId is roughly the billing account for the Microsoft account.  id is the uuid for the secureScore report.  likely each generation of the report would have a different id.

Reply:
azureTenantId is roughly the billing account for the Microsoft account. id is the uuid for the secureScore report. likely each generation of the report would have a different id.

Reply:
Makes sense.  Maybe the uuids for the securescore report should be concatenated and put into the profile name or something?